compiler/passes/ResolveScope.cpp

```diff
@@ -665,7 +661,8 @@ Symbol* ResolveScope::lookupForImport(Expr* expr, bool isUse) const {
       USR_PRINT(expr, "please specify the full path to the module");
       USR_PRINT(expr, "or use a relative %s e.g. '%s this.M' or '%s super.M'",
                       stmtType, stmtType, stmtType);
-      USR_STOP();
+      // Try to continue compilation
+      retval = badCloserModule;
```

Comment:
Should this still be here, or was it a debugging aid?

Reply:
It should still be here - it's about producing multiple such errors.